components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -945,42 +945,47 @@ func runIDEReadinessProbe(cfg *Config, ideConfig *IDEConfig, ide IDEKind) (deskt
 			port   = defaultIfZero(ideConfig.ReadinessProbe.HTTPProbe.Port, defaultProbePort)
 			url    = fmt.Sprintf("%s://%s:%d/%s", schema, host, port, strings.TrimPrefix(ideConfig.ReadinessProbe.HTTPProbe.Path, "/"))
 			client = http.Client{Timeout: 1 * time.Second}
-			tick   = time.NewTicker(500 * time.Millisecond)
+			tick   = time.NewTicker(250 * time.Millisecond)
 		)
 		defer tick.Stop()
 
 		t0 := time.Now()
 
-		for {
-			<-tick.C
-
+		for ; true; <-tick.C {
```

Comment:
I think this doesn't read nice, how about `for _ := range tick.C` ?

Reply:
@jeanp413 that will wait for a tick. That said, if you prefer that, I can change the code

Reply:
isn't `for ; true; <-tick.C` also waiting for the tick? if it's not waiting then why we need the tick at all :thinking:

Reply:
changed